tests/e2e/vmgroup_volumeops_test.go

```diff
@@ -0,0 +1,271 @@
+		t.Errorf("Expected error when attaching volume %s in default  from VM %s in vmgroup %s", volName1, DockerHosts[0], Vmgroup)
+	}
+
+	// 3. Try deleting volume in default group
+	_, err = dockercli.DeleteVolume(DockerHosts[0], volName1)
+	if err == nil {
+		t.Errorf("Expected error when deleting volume %s in default  from VM %s in vmgroup %s", volName1, DockerHosts[0], Vmgroup)
+	}
+
+	// 4. Remove the volume from the default , from the other VM
+	_, err = dockercli.DeleteVolume(DockerHosts[1], volName1)
+	if err != nil {
+		t.Errorf("Error when deleting volume %s in default  from VM %s in vmgroup %s", volName1, DockerHosts[1], Vmgroup)
+	}
+
+	// 5. Remove the VM from the new vmgroup
+	cmd = admincli.RemoveVMsForVmGroup + Vmgroup  + " --vm-list " + DockerHosts[0]
+	_, err = ssh.ExecCmd(Host, cmd)
+	if err != nil {
+		t.Errorf("Error when removing VM %s from  on host %s - [%s], err %s", DockerHosts[1], Host, cmd, err.Error())
+	}
+
+	// 6. Remove the new 
+	cmd = admincli.DeleteVmGroup + Vmgroup
+	_, err = ssh.ExecCmd(Host, cmd)
+	if err != nil {
+		t.Errorf("Error when removing vmgroup [%s] on host %s - [%s], err %s", DockerHosts[1], Host, cmd, err.Error())
+	}
+
+	t.Logf("Passed - Volume access across s")
+}
+
+// TestCreateAccessPrivilegeOnDefaultVg - Verify volumes can be
+// created by a VM as long as the  has the allow-create setting
+// enabled on it
+func TestCreateAccessPrivilegeOnDefaultVg(t *testing.T) {
+	// Create a volume in the default group
+	createVolumeOnDefaultVg(t, volName1)
+
+	// 1. Attach volume from default 
+	_, err := dockercli.AttachVolume(DockerHosts[0], volName1, Cname)
+	if err != nil {
+		t.Errorf("Error while attaching volume - %s on VM - %s, err - %s", volName1, DockerHosts[0], err.Error())
+	}
+
+	// 2. Remove the create privilege on the default  for specified datastore
+	cmd := admincli.ModifyAccessForVmGroup + defaultVg + "--allow-create False --datastore " + Ds1
+	_, err = ssh.ExecCmd(Host, cmd)
+	if err != nil {
+		t.Errorf("Error when setting access privileges [%s] on default  on host %s, err - %s", cmd, Host, err.Error())
+	}
+
+	// 3. Try creating a volume on the default 
+	_, err = dockercli.CreateVolume(DockerHosts[0], volName2)
+	if err == nil {
+		t.Errorf("Expected error while creating volume - %s from VM - %s, on default , err - %s", volName2, DockerHosts[0], err.Error())
+	}
+
+	// 4. Restore the create privilege on the default  for specified datastore
+	cmd = admincli.ModifyAccessForVmGroup + defaultVg + "--allow-create True --datastore " + Ds1
+	_, err = ssh.ExecCmd(Host, cmd)
+	if err != nil {
+		t.Errorf("Error when restoring access privileges [%s] on default  on host %s, err - %s", cmd, Host, err.Error())
+	}
+
+	// 5. Remove the volume created earlier
+	_, err = dockercli.DeleteVolume(DockerHosts[0], volName1)
+	if err != nil {
+		t.Errorf("Error while deleting volume - %s in the default , err - %s", volName1, err.Error())
+	}
+	t.Logf("Passed - create privilege on default ")
```

Comment:
In existing tests, we are adding 2 tracing logs for each test:

log.Printf("START: <test-group>.<test-case>")
log.Printf("END: <test-group>.<test-case>")

Can we follow the same so that we have a consistent output?